config/setup_consul_no_secrets.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
```

Comment:
So this file replaces `settings.sh`? As in for development / deployment we should make `setup_consul_dev.sh`, etc?